docs/content/reference/upgrade_dependencies.html.markdown

```diff
@@ -0,0 +1,18 @@
+# Upgrading Dependencies
+
+## Procedure
+
+1. Please have an instance of `magic-modules` available. It will be used later in this procedure.
+2. Checkout & `cd` to [`terraform-provider-google-beta`](http://github.com/hashicorp/terraform-provider-google-beta)
+3. Upgrade your desired package. EX: `go get -u google.golang.org/api` You may need to upgrade all packages at once with `go get -u ./..`. [More Info](https://golang.cafe/blog/how-to-upgrade-golang-dependencies.html).
```

Comment:
See https://developers.google.com/style/format-examples for style guidance on examples, and https://developers.google.com/style/procedures for information about how to format procedure lists that include code.

Reply:
i hope i improved it.